src/serial/SerialInputHandler.cpp

```diff
@@ -515,6 +537,8 @@ domain                 get backend domain
 domain       <domain>  set backend domain
 authtoken              get auth token
 authtoken    <token>   set auth token
+lcg_override           get LCG override
+lcg_override <domain>  set LCG override
```

Comment:
Nitpick:

Not sure about the underscore in the command name, since each other command with multiple words/acronyms (`authtoken`, `rftransmit`, `factoryreset`, `rftxpin`) is all one word, no special characters. Strange to break convention here.